### PR TITLE
Added feature to ping a given mantainer per repo for dependabot PRs

### PR DESCRIPTION
**Pull request by lluiscampos**
Used in the Mender team where each subteam (frontend, backend and
    client) has a single person responsible for dependabot reviews and
    updates.
    
    The feature is enabled only for `mender-test-bot` bot and has hard-coded
    maintainers per repository. We will modify it as we rotate the role.

### CODE REVIEW

File: tom/bot.py [hunk 1]
```diff
@@ -304,13 +305,24 @@ def find_reviewers(self, pr):
...
-        url = pr["url"].replace("https://api.github.com/repos/", "")
         log.info("Looking at: {} ({})".format(pr["title"], pr["html_url"]))
 
         pr = PR(pr, self.github)
         if "ping_reviewers" in self.bot_features:
             self.find_reviewers(pr)
+        if "ping_reviewers_dependabot" in self.bot_features:
+            self.assign_dependabot_maintainer(pr)
+        if "ping_reviewers" in self.bot_features or "ping_reviewers_dependabot" in self.bot_features:
             self.ping_reviewer(pr)
```
Comment (olehermanse) [inline tom/bot.py:326]: So, as I pointed out face to face: You should set a default vaue for `reviewer` in `class PR` (`github.py`). Then in `ping_reviewer()` you should add an if to skip if there is no reviewer found.

**Review by olehermanse — CHANGES_REQUESTED** [review]
Looks good so far, only one blocker I see.

**Comment by olehermanse** [comment]
@lluiscampos Regarding CI, don't worry I'll fix.

File: tom/bot.py (hunk 1 above)
Comment (lluiscampos) [reply tom/bot.py:333]: Thanks. I fixed that now.

**Comment by lluiscampos** [comment]
I also moved the feature to the mender bot part and amended the commit message accordingly :+1:

File: config.json [hunk 2]
```diff
@@ -66,8 +66,51 @@
...
+        "mendersoftware/iot-manager": "kjaskiewiczz",
+        "mendersoftware/mender-configure-module": "kjaskiewiczz",
+        "mendersoftware/monitor-client": "kjaskiewiczz",
+        "mendersoftware/mtls-ambassador": "kjaskiewiczz",
+        "mendersoftware/reporting": "kjaskiewiczz",
+        "mendersoftware/tenantadm": "kjaskiewiczz",
+        "mendersoftware/useradm": "kjaskiewiczz",
+        "mendersoftware/useradm-enterprise": "kjaskiewiczz",
+        "mendersoftware/workflows": "kjaskiewiczz",
+        "mendersoftware/workflows-enterprise": "kjaskiewiczz"
```
Comment (olehermanse) [inline config.json:112]: One idea here, if we want to ensure we cover all the repos, is to set a default (Lluis or Krzysztof) and then only list the repos which should have a different reviewer.

**Comment by olehermanse** [comment]
@lluiscampos Small FYI; we don't use `chore: ` and `feat: ` in CFEngine repos. In this one, it doesn't matter much, but in other python repos like `cfbs` and `cf-remote` we use PR titles to generate changelogs (GitHub feature), so I'd tell you to remove it / stop doing it there.

**Review by olehermanse — APPROVED** [review]
I've made my own fixes and tested this manually, seems to work! Thank you for working on this 👍

**Review by cf-bottom — APPROVED** [review]
I trust @olehermanse, approved!

**Comment by lluiscampos** [comment]
> @lluiscampos Small FYI; we don't use `chore: ` and `feat: ` in CFEngine repos. In this one, it doesn't matter much, but in other python repos like `cfbs` and `cf-remote` we use PR titles to generate changelogs (GitHub feature), so I'd tell you to remove it / stop doing it there. (See: https://github.com/cfengine/core/blob/master/CONTRIBUTING.md)

Thank you. I removed the prefixes now :+1:

File: config.json (hunk 2 above)
Comment (lluiscampos) [reply config.json]: Originally I wanted something like that: define the backend, client and frontend maintainers and then map repos to team. But I realized that the maintenance effort would be the same as we sill need to add repos to one of the groups any time we have a new repo. So I concluded in having a directy repo to maintainer list.

**Pull request merged**